src/dist/http/test/http_server_test.cpp

```diff
@@ -27,7 +27,8 @@ TEST(http_server, parse_url)
         {"http://127.0.0.1:34601/threads/", ERR_OK, {"threads", ""}},
         {"http://127.0.0.1:34601//pprof/heap/", ERR_OK, {"pprof", "heap"}},
         {"http://127.0.0.1:34601//pprof///heap", ERR_OK, {"pprof", "heap"}},
-        {"http://127.0.0.1:34601/pprof/heap/arg/", ERR_INVALID_PARAMETERS, {}},
+        {"http://127.0.0.1:34601/pprof/heap/arg/", ERR_OK, {"pprof", "heap/arg"}},
+        {"http://127.0.0.1:34601/pprof///heap///arg/", ERR_OK, {"pprof", "heap/arg"}},
```

Comment:
Why there are so many `/`? What's the different with a single `/`?

Reply:
I test `multiple slashes` in browser, It is same to `single slash`.